src/sgame/sg_cmds.cpp

```diff
@@ -3497,6 +3497,98 @@ void Cmd_Reload_f( gentity_t *ent )
 	}
 }
 
+/*
+=================
+Cmd_TeamStatus_f
+=================
+*/
+void Cmd_TeamStatus_f( gentity_t * ent )
+{
+	int builders = 0;
+	gentity_t *tmp;
+	struct buildable_status_t
+	{
+		int count;
+		int health;
+		bool spawned;
+	} structures[ BA_NUM_BUILDABLES ] = {};
+
+	if ( !g_teamStatus.Get() ) 
```

Comment:
This is an integer, so you likely want ` < 0` here.

Reply:
done